pkgs/development/python-modules/torchvision/bin.nix

```diff
@@ -26,10 +28,17 @@ in buildPythonPackage {
 
   src = fetchurl srcs."${stdenv.system}-${pyVerNoDot}" or unsupported;
 
-  disabled = !(isPy37 || isPy38 || isPy39 || isPy310);
+  disabled = !(isPy38 || isPy39 || isPy310 || isPy311);
```

Comment:
nit: using pythonAtLeast and pythonOlder could be nicer

Reply:
Fixed :)